build_script.py

```diff
@@ -184,40 +185,60 @@ def main(args=sys.argv[1:]):
     function.
     """
     parser = argparse.ArgumentParser(
-        description="Builds, tests, and installs XCTest.")
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+        description=textwrap.dedent("""
+            Build, test, and install XCTest.
+
+            WARNING: This script is only meant to be used on Linux host
```

Comment:
I'd suggest that it would be better to refer to "Linux environments" here rather than "Linux host machines", since that term is a bit overloaded, and many devs referring to this are likely to be running their Linux on an _OS X host machine_.